pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -192,7 +192,7 @@ func registerAWSDMS(r registry.Registry) {
 		Owner:   registry.OwnerMigrations,
 		Cluster: r.MakeClusterSpec(1),
 		Leases:  registry.MetamorphicLeases,
-		Tags:    registry.Tags(`default`, `awsdms`, `aws`),
+		Tags:    registry.Tags(`default`, `awsdms`, `aws`, `weekly`),
```

Comment:
I think the only 2 tags required here are `weekly` and `aws-weekly`. The former gets special treatment in roachtest [1], and the latter is required to actually run it in the weekly aws job.

[1] - [test_registry.go](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/test_registry.go#L149)
[2] - [roachtest_weekly_aws_impl.sh](https://github.com/cockroachdb/cockroach/blob/master/build/teamcity/cockroach/nightlies/roachtest_weekly_aws_impl.sh#L19)

Reply:
thanks! amended.